java/src/ApiImpl.java

```diff
@@ -15,27 +18,77 @@ public abstract class ApiImpl implements AuthenticatedApi{
     private Vault vault;
 
     public List<Reservation> getMyReservations(int userId){
```

Comment:
What is the purpose of this code? If this is meant to return the reservation of any user, we should probably change the name, and maybe validate that is the requester is an admin, or we should use the id of the logged in user